data/src/main/kotlin/land/sungbin/androidprojecttemplate/data/ChatRoom.kt

```diff
@@ -0,0 +1,26 @@
+package land.sungbin.androidprojecttemplate.data
+
+/**
+ * 채팅방[ChatRoom] data class
+ *
+ * @param isDeleted 삭제 여부
+ * @param chatRoomType [ChatRoomType] 채팅 유형
+ * @param field 채팅방 필드 값
+ * @param chatCategory [LikeCategory] 카테고리
+ * @param tags 태그 목록
+ */
+data class ChatRoom(
+    val isDeleted: Boolean,
+    // TODO (riflockle7) serializer 로 int 와 대응하도록 처리 필요
+    val chatRoomType: ChatRoomType,
```

Comment:
클래스명부터 ChatRoom 인데 굳이 chatRoom 을 명시해줄 필요가 있을까요?

```suggestion
    val type: ChatRoomType,
```

Reply:
놓친 내용 짚어주셨군요 👍